data-raw/prepare_cached_habitat.R

```diff
@@ -382,4 +392,6 @@ low_mid_sac_fp <- cvpiaHabitat::set_floodplain_habitat('Lower-mid Sacramento Riv
 
 wr_fp[21,,] <- low_mid_sac_fp
 
-usethis::use_data(wr_fp)
+wr_fp[is.na(wr_fp)] <- 0
```

Comment:
not needed, right?

Reply:
it looks like the winter run (floodplain, juv and fry) are not calling the the habitat functions in this file instead they are calling cvpiaHabitat::set_instream_habitat directly 

```
wr_fp <- array(NA, c(31, 12, 20))
wr_fp[1,,] <- cvpiaHabitat::set_floodplain_habitat('Upper Sacramento River', 'wr', get_flow('Upper Sacramento River'))
wr_fp[16,,] <- cvpiaHabitat::set_floodplain_habitat('Upper-mid Sacramento River', 'wr', get_flow('Upper-mid Sacramento River'))
wr_fp[24,,] <- cvpiaHabitat::set_floodplain_habitat('Lower Sacramento River', 'wr', get_flow('Lower Sacramento River'))

# lower-mid sacramento 
low_mid_sac_flows1 <- get_flow("Lower-mid Sacramento River1") 
low_mid_sac_flows2 <- get_flow("Lower-mid Sacramento River2") 
low_mid_sac_fp <- cvpiaHabitat::set_floodplain_habitat('Lower-mid Sacramento River', 'wr',
                                                       low_mid_sac_flows1, flow2 = low_mid_sac_flows2)

wr_fp[21,,] <- low_mid_sac_fp
```

Reply:
good point!